cmd/keytransparency-server/main.go

```diff
@@ -121,8 +121,9 @@ func main() {
 	tmap := trillian.NewTrillianMapClient(mconn)
 
 	// Create gRPC server.
+	var revisionPageSize int32 = 10
```

Comment:
Would this not be better as a flag?

Reply:
I'm trying to only have flags in the main.go file.

Reply:
This is main.go

Reply:
🤦🏻‍♂️ - Fixed